swap/src/bob/swap.rs

```diff
@@ -181,11 +181,19 @@ where
                     let cancel_timelock_expires =
                         state3.wait_for_cancel_timelock_to_expire(bitcoin_wallet.as_ref());
 
+                    // Record the current monero wallet block height so we don't have to scan from
```

Comment:
People with a wallet few years old would still have an issue, could you go for the correct solution to pass the lock transaction height?

Reply:
I don't understand what you mean. We are creating a new wallet every time and pass the `restore_height` to it. I don't understand how this involves "People with a wallet few years old ".

Reply:
My understanding was incorrect :) thanks.